provisioning/miniprov/hare_mp/cdf.py

```diff
@@ -37,11 +38,68 @@ def _create_node_descriptions(self) -> List[NodeDesc]:
             nodes.append(self._create_node(node))
         return nodes
 
+    def _create_pool_descriptions(self) -> List[PoolDesc]:
+        pools: List[PoolDesc] = []
+        conf = self.provider
+        cluster_id = conf.get('cluster>cluster_id')
```

Comment:
I don't see `cluster>cluster_id` but there is `server_node>{machine-id}>cluster_id` in [conf-store-keys](https://seagatetechnology.sharepoint.com/:x:/r/sites/gteamdrv1/tdrive1224/_layouts/15/Doc.aspx?sourcedoc=%7B46F15394-2BA7-4D44-8B7E-67F1AEAEA654%7D&file=LDR%20R2%20Specifications%20-%20Provisioning.xlsx&action=default&mobileredirect=true&cid=c993b23a-e712-46a5-8d38-ca987a646b31)

Reply:
cluster_id is available at multiple path (e.g. server_node>{machine-id}>cluster_id, storage_enclosure>{enclosure-id}>cluster_id, etc.).
Also key I used is present in json file currently used in deployment and looks more logical than other places as it is at cluster level.
http://eos-jenkins.colo.seagate.com/job/Cortx-Main/job/centos-7.8.2003/job/Main%20Deploy/88/artifact/provisioner_cluster.json/*view*/

Let me know your thoughts.

Note: Not specific to this but in general, I am assuming that all the keys which are defined and used by us for single node cluster will be still valid and addition of new keys mentioned in 'ConfStore Keys (N-Node)'

Reply:
Modified code to use 'server_node>{machine-id}>cluster_id'

Reply:
> Not specific to this but in general, I am assuming that all the keys which are defined and used by us for single node cluster will be still valid after addition of new keys mentioned in 'ConfStore Keys (N-Node)'

Not necessarily, I see they have changed the keys altogether.

Reply:
It is possible that CI may fail until we have the correct conf-store file. But that is fine, we need to fix that.
Deployment failure is also expected but that too needs to be fixed by the provisioning team. I will confirm this.